bundles/org.openhab.ui.basic/src/main/java/org/openhab/ui/basic/internal/render/ImageRenderer.java

```diff
@@ -98,7 +98,7 @@ public boolean canRender(Widget w) {
         snippet = StringUtils.replace(snippet, "%proxied_url%", proxiedUrl);
         snippet = StringUtils.replace(snippet, "%ignore_refresh%", ignoreRefresh ? "true" : "false");
         snippet = StringUtils.replace(snippet, "%url%", url);
-
+        // logger.warn(snippet);
```

Comment:
To be suppressed with logger variable definition.
Other option: use a unvommented TRACE log but add something to your log like "snippet for image widget : {}"

Reply:
Removed.